packages/@uppy/companion/src/server/controllers/s3.js

```diff
@@ -16,6 +24,10 @@ module.exports = function s3 (config) {
     throw new TypeError('s3: The `getKey` option must be a function')
   }
 
+  function getBucket (req) {
+    return typeof config.getBucket === 'function' ? config.getBucket(req) : config.bucket
```

Comment:
shouldn't we use `config.bucket` instead of `config.getBucket`? then it can be either a string or a function, and it will be backwards compatible